src/Kusto/examples/Add-AzKustoDatabasePrincipal.md

```diff
@@ -1,9 +1,7 @@
 ### Example 1: Add Database principals permissions
 ```powershell
-Add-AzKustoDatabasePrincipal -ResourceGroupName testrg -ClusterName testnewkustocluster -DatabaseName mykustodatabase -Value (@{Name="Some User"; Role="Admin"; Type="User"; Email="someuser@microsoft.com"})
-```
+PS C:\> Add-AzKustoDatabasePrincipal -ResourceGroupName testrg -ClusterName testnewkustocluster -DatabaseName mykustodatabase -Value (@{Name="Some User"; Role="Admin"; Type="User"; Email="someuser@microsoft.com"})
 
-```output
```

Comment:
Please keep the `output` block and remove the `PS C:\> ` in all `powershell` block. This is our new format.